mobile/src/main/java/org/openhab/habdroid/background/ItemUpdateWorker.kt

```diff
@@ -42,18 +41,20 @@ class ItemUpdateWorker(context: Context, params: WorkerParameters) : Worker(cont
             }
         }
 
-        val item = data.getString(INPUT_DATA_ITEM)
+        val item = inputData.getString(INPUT_DATA_ITEM)
         val value = inputData.getString(INPUT_DATA_VALUE) as String
+        val successToastMessage = inputData.getString(INPUT_DATA_SUCCESS_TOAST_MESSAGE)
 
         return runBlocking {
             try {
                 val result = connection.httpClient
                     .post("rest/items/$item", value, "text/plain;charset=UTF-8")
                     .asStatus()
                 Log.d(TAG, "Item '$item' successfully updated to value $value")
+                successToastMessage?.let { Util.showToast(applicationContext, it) }
```

Comment:
I think I'd prefer an `if` check instead of `let` for clarity.

(I usually only use `let` for mutable variables)